201-vmss-windows-webapp-dsc-autoscale/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "value": "Standard_A1"
+    },
+    "windowsOSVersion": {
+      "value": "2012-R2-Datacenter"
+    },
+    "vmssName": {
+      "value": "GEN-UNIQUE-5"
+    },
+    "instanceCount": {
+      "value": 3
+    },
+    "adminUsername": {
+      "value": "vmssadmin"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "_artifactsLocation": {
+      "value": null
```

Comment:
Instead of having null values for parameters with defaults, it's better to simply not have these parameters listed in the parameters file (there is a bug in the CLI which caused the error you were seeing, but that's for them to fix)